packages/ca_certificates/project.bri

```diff
@@ -53,7 +53,7 @@ export function liveUpdate(): NushellRunnable {
     let version = http get https://curl.se/docs/caextract.html
       | lines
       | where ($it | str contains 'href="/ca/cacert-')
-      | parse --regex '<a href="/ca/cacert-(?<version>.+).pem"'
+      | parse --regex '<a href="/ca/cacert-(?<version>.+)\.pem"'
```

Comment:
Should be double-escaped

```suggestion
      | parse --regex '<a href="/ca/cacert-(?<version>.+)\\.pem"'
```

(double escaping always throws me off a bit... in this case, I validated it by using `brioche-packer` from [`brioche-runtime-utils`](https://github.com/brioche-dev/brioche-runtime-utils) to check what the escapes looked like in the raw script):

```sh
brioche build -p packages/ca_certificates -e liveUpdate -o _local/output --replace
brioche-packer read _local/output/brioche-run | jq '.metadata | fromjson.args[1].value.components[0].value' -r
```